apps/api/src/app/workflows-v2/mappers/notification-template-mapper.ts

```diff
@@ -63,10 +63,22 @@ function toMinifiedWorkflowDto(template: NotificationTemplateEntity): WorkflowLi
     updatedAt: template.updatedAt || 'Missing Updated At',
     stepTypeOverviews: template.steps.map(buildStepTypeOverview).filter((stepTypeEnum) => !!stepTypeEnum),
     createdAt: template.createdAt || 'Missing Create At',
-    status: WorkflowStatusEnum.ACTIVE,
+    status: getWorkflowStatus(template),
   };
 }
 
+function getWorkflowStatus(template: NotificationTemplateEntity): WorkflowStatusEnum {
+  if (template.issues && Object.keys(template.issues).length > 0) {
+    return WorkflowStatusEnum.ERROR;
+  }
+
+  if (template.active) {
+    return WorkflowStatusEnum.ACTIVE;
+  }
+
+  return WorkflowStatusEnum.INACTIVE;
+}
+
```

Comment:
I think this state should come from the backend, we shouldn't be computing the status on the frontend.

Reply:
true it is computed once we know that issue,
PostProcessWorkflowUpdate.overloadStatusOnWorkflow

Reply:
Please disregard. I read `dashboard` in the PR title, it should be `feat(dashboard, api)` to signify changes in both.